CMakeLists.txt

```diff
@@ -50,7 +50,13 @@ set(SONATA_SRC
     src/hdf5_mutex.cpp
     src/nodes.cpp
     src/population.cpp
-)
+    src/version.cpp
```

Comment:
I feel a bit leery about writing intermediate files to `src`.
Should it be picked up from `build` instead?

Reply:
I tried but did not managed to. I can try harder if you want

Reply:
Ok, got it

Reply:
no need for ` ${CMAKE_CURRENT_BINARY_DIR}` here?